hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -1032,11 +1035,11 @@ public static void accumulateColumnRanges(Schema.Field field, String filePath,
                                             Map<String, HoodieColumnRangeMetadata<Comparable>> columnRangeMap,
```

Comment:
Let's move `accumulateColumnRanges` closer to where it's actually used (into `HoodieAppendHandle`)